### ISSUE TITLE
sqlite3.dll is not included in installer yet

### ISSUE THREAD

**Issue by cbielow** [OpenMS/windows-installer#13]
it should be added, since exectuables require it to run :)

**Comment by hroest** [comment]
indeed, sounds important - I guess also coinor which we recently added to the build....

**Comment by cbielow** [comment]
fixed in https://github.com/OpenMS/windows-installer/commit/6ae4967147e3de10e78ad2e12f20f8c2c63cb2fa

@jpfeuffer is coinor statically linked? I.e. no dll required?

**Comment by jpfeuffer** [comment]
Yes CoinOR was in there for a long time (statically). It just is now actually used as default instead of GLPK.
